igv_remote/_core.py

```diff
@@ -9,19 +9,33 @@
 
 def _append_id(filename, id):
     return "{0}_{2}.{1}".format(*filename.rsplit('.', 1) + [id])
-        
+
+def _parse_loc(chromosome, pos1, pos2=None, expand=20):
+    if expand < 20:
+        print("IGV expands left and right margin by at least 20bp")
+        expand=20
+    start_pos = int(pos1-expand)
+    if pos2 is None:
+        end_pos = int(pos1+expand)
+    else:
+        end_pos = int(pos2)
+    start_pos ='{:,}'.format(int(start_pos))
+    end_pos = '{:,}'.format(int(end_pos))
+    position= 'chr{}:{}-{}'.format(int(chromosome),start_pos, end_pos)
```

Comment:
Down the road, we should create a `convert_chr` function (from Capy?) that automatically parses the various formats people use to supply chromosome names. This is fine for now, but it isn't safe to assume that people will provide chromosomes as integers.

The formats we'll need to handle are:

chr[0-9XY]+
[0-9XY]+
[0-9]+